src/pages/Profile.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import './Profile.css';
+
+const Profile = () => {
+  const rockets = useSelector((state) => state.rockets);
+  const missions = useSelector((state) => state.missions);
+  const rocketElement = rockets.map((rocket) => (
+    rocket.active === true ? <li className="rockets">{rocket.rocket_name}</li> : null));
+  const missionElement = missions.map((mission) => (
```

Comment:
- [ ] **Required:** Kindly fix these console errors which are thrown as rockets and missions are mapped over. These are because you have not provided any keys.

![image](https://user-images.githubusercontent.com/27709832/175550545-e2d36583-5548-4c38-a3e7-40cbead9ba59.png)